flask_menu/__init__.py

```diff
@@ -90,14 +97,19 @@ def _active_when(self):
 
     def register(self, endpoint, text, order=0,
                  endpoint_arguments_constructor=None,
+                 expected_args=None,
```

Comment:
just for better backward compatibility, can you move it after `visible_when` argument?